test/e2e/egressip.go

```diff
@@ -2525,4 +2525,102 @@ spec:
 			podNamespace.Name, true, []string{pod1Node.nodeIP}))
 		framework.ExpectNoError(err, "7. Check connectivity to the host on the OVN network from the pod not selected by EgressIP, failed: %v", err)
 	})
+
+	// Single EgressIP object where the Egress IP of object is hosted on a single interface thats enslaved to a VRF device on a secondary host network
+	// 0. create VRF and enslave expected egress interface
```

Comment:
nice. I guess we could make this a little more robust in the future by having multiple interfaces with different routes, and verify routing only flows via the VRF routes for egress IP. I think in this test case you are just moving the regular egress interface into a VRF. Can do it in a later PR if you want.